bears/scss/SCSSLintBear.py

```diff
@@ -21,4 +23,273 @@ class SCSSLintBear:
+            ``camel_case``, ``snake_case``), or a ``regex`` the name must match
+            (eg: ``^[a-zA-Z]+$``) to use for variable.
+        :param placeholder_naming_convention:
+            Name of convention (``hyphenated_lowercase`` (default),
+            ``camel_case``, ``snake_case``), or a ``regex`` the name must match
+            (eg: ``^[a-zA-Z]+$``) to use for placeholder.
+        :param max_depth:
+            Maximum nesting depth.
+        :param use_placeholder_selector_in_extend:
+            Always use placeholder selectors in ``@extend``.
+        :param max_properties:
+            Limit the number of properties in a rule set.
+        :param allow_unit_on_zero_values:
+            Omit length units on zero values.
+        :param check_urls_format:
+            URLs should be valid and not contain protocols or domain names.
+        :param urls_in_quotes:
+            URLs should always be enclosed within quotes.
+        :param allow_unnecessary_parent_reference:
+            Allows use of the parent selector references ``&`` even when they
+            are not unnecessary.
+        :param allow_unnecessary_mantissa:
+            Numeric values can contain unnecessary fractional portions.
+        :param allow_traling_whitespaces:
+            Reports lines containing trailing whitespace.
+        :param allow_traling_semicolon:
+            Property values; ``@extend``, ``@include``, and ``@import``
+            directives; and variable declarations should always end with a
+            semicolon.
+        :param check_imports_path:
+            The basenames of ``@import``ed SCSS partials should not begin with
+            an underscore and should not include the filename extension.
+            These requirements can be modified by changing
+            ``allow_filename_leading_underscore``, and ``allow_extensions``.
+        :param allow_filename_leading_underscore:
+            Allows basenames of ``@import``ed SCSS partials to begin with an
+            underscore.
+        :param allow_extension:
+            Allows basnames of ``@import``ed SCSS partials to include filename
+            extension.
+        :param use_length_variables:
+            Prefer length literals (numbers with units) to be used only in
+            variable declarations.
+        :param check_properties_spelling:
+            Reports when an unknown or disabled CSS property is used
+            (ignoring vendor-prefixed properties).
+        :param extra_properties:
+            List of extra properties to allow.
+        :param disabled_properties:
+            List of existing properties to deny.
+        :param check_pseudo_elements:
+            Pseudo-elements, like ``::before``, and ``::first-letter``,
+            should be declared with two colons. Pseudo-classes, like ``:hover``
+            and ``:first-child``, should be declared with one colon.
+
+            ``
```

Comment:
No ... double backticks don't work for code blocks ... Have you tested this out ? Double backticks is only for single line codes

This gives more info... http://www.sphinx-doc.org/en/stable/markup/code.html